src/App.css

```
@@ -355,3 +355,20 @@
 .btn-border:hover {
   background-color: #ecf0f4;
 }
+
+.inputForm-btnGroup {
+  display: flex;
+  justify-content: space-between;
+  padding-top:15px
+}
+
+.btn-InputModalSuccess{
```

naming should be consistent
```suggestion
.btn-Input-modal-success{
```